client/src/services/activity.ts

```
@@ -154,3 +154,19 @@ export const getComments = async (activityID: string) => {
     console.error(error);
   }
 };
+
+export const deleteActivity = async (username: string, activityID: string) => {
+  try {
+    const response = await fetch(
+      `${BASE_URL}/delete-activity/${username}/${activityID}`,
+      {
+        method: 'DELETE',
+      }
+    );
+    if (!response.ok) console.error('ERROR: ', response);
```

we need to throw an error here in order to catch it in the `catch` block